.github/workflows/linux-tests.yml

```
@@ -0,0 +1,124 @@
+name: GitHub Actions CI - (Linux)
+on:
+  pull_request:
+    paths-ignore:
+      # Slow running processes that will be built on the self-hosted runner
+      - "Formula/bison.rb"
+      - "Formula/bzip2.rb"
+      - "Formula/curl.rb"
+      - "Formula/expat.rb"
+      - "Formula/flex.rb"
+      - "Formula/libffi.rb"
+      - "Formula/libxml2.rb"
+      - "Formula/libxslt.rb"
+      - "Formula/libx11.rb"
+      - "Formula/m4.rb"
+      - "Formula/ncurses.rb"
+      - "Formula/perl.rb"
+      - "Formula/python@3.9.rb"
+      - "Formula/sqlite.rb"
+      - "Formula/zlib.rb"
+      # Will be build in the linux-pr-tests-wheezy workflow
```

```suggestion
      # Will be built in the linux-pr-tests-wheezy workflow
```
Should those also be called `linux-tests-wheezy`?